### PR TITLE
Install Google Cloud Logging and Monitoring when installing on GCE.

### PR DESCRIPTION
**Pull request by ewiseblatt**
@lwander 

This has to be explicitly requested with arguments to InstallSpinnaker.

The build_google_image script builds both these in by default, but an explicit
install (calling InstallSpinnaker.sh directly) would not include these by default even on GCE.

### CODE REVIEW

**Comment by ewiseblatt** [comment]
This can be reviewed and merged now.

File: InstallSpinnaker.sh [hunk 1]
```diff
@@ -311,6 +328,43 @@ function install_java() {
...
+        || [[ "x$GOOGLE_CLOUD_MONITORING" == "xtrue" ]]; then
+          if [[ ! -f /etc/google/auth/application_default_credentials.json ]];
+          then
+            echo "You may need to add Google Project Credentials."
+            echo "See https://developers.google.com/identity/protocols/application-default-credentials"
+          fi
+     fi
+  fi
+
+  if [[ "x$GOOGLE_CLOUD_LOGGING" == "xtrue" ]]; then
```
Comment (lwander) [inline InstallSpinnaker.sh]: Why do you prefix both sides with "x"?

**Comment by lwander** [comment]
LGTM

File: InstallSpinnaker.sh (hunk 1 above)
Comment (ewiseblatt) [inline InstallSpinnaker.sh]: I was following the practices elsewhere in the file. It's archaic and shouldnt be needed these days. in fact, it's probably only the -z test that [would have] needed it.

**Pull request merged**